litmus-portal/backend/graphql-server/pkg/graphql/mutations/mutation.go

```diff
@@ -0,0 +1,85 @@
+package mutations
```

Comment:
I moved all the mutation related functions from pkg/cluster to pkg/graphql/mutations, because every file contains only one function and again single function in a file doesn't make sense.
If this file grows, we need to think for the modularity of mutation.go

Reply:
Yes I made it in separate files for the ease of locating the particular function, but having it in one file makes sense.

Reply:
If we put one function in a single file, the files inside a directory will grow. which is not a good approach, we need to generalize the functions as much as possible.